docs/reference/monitoring/index.asciidoc

```diff
@@ -5,38 +5,42 @@
 
 [partintro]
 --
-{monitoring} enables you to easily monitor the health of your {es} cluster. The
-monitoring metrics are collected from each node and stored in {es} indices.
+The Elastic {monitor-features} enable you to easily monitor the health of 
+your {es} cluster. The monitoring metrics are collected from each node and 
+stored in {es} indices.
 
 Each {es} node is considered unique based on its persistent UUID, which is
 written on first start to its <<path-settings,`path.data`>> directory, which
 defaults to `./data`.
 
-All settings associated with {monitoring} in {es} must be set in either the
+All settings associated with monitoring in {es} must be set in either the
 `elasticsearch.yml` file for each node or, where possible, in the dynamic
 cluster settings. For more information, see <<configuring-monitoring>>.
 
 [[es-monitoring-overview]]
-{es} is also at the core of {monitoring} across the Elastic Stack. In all cases, 
-{monitoring} documents are just ordinary JSON documents built by monitoring each 
-Elastic Stack component at some collection interval, then indexing those 
-documents into the monitoring cluster. Each component in the stack is 
-responsible for monitoring itself and then forwarding those documents to {es}
-for both routing and indexing (storage).
+{es} is also at the core of monitoring across the {stack}. In all cases, 
+monitoring documents are just ordinary JSON documents built by monitoring each 
+{stack} component at some collection interval, then indexing those 
+documents into the monitoring cluster. 
 
+Each component in the stack is responsible for monitoring itself and then 
+forwarding those documents to {es} for both routing and indexing (storage).
 The routing and indexing processes in {es} are handled by what are called
-<<es-monitoring-collectors,collectors>> and 
-<<es-monitoring-exporters,exporters>>. In the past, collectors and exporters
-were considered to be part of a monitoring "agent", but that term is generally
-not used anymore.
+<<es-monitoring-collectors,collectors>> and <<es-monitoring-exporters,exporters>>. 
+In the past, collectors and exporters were considered to be part of a monitoring 
+"agent", but that term is generally not used anymore. 
+
+beta[] Alternatively, in 6.4 and later, you can use {metricbeat} to collect and 
+ship data about {kib}, rather than routing it through {es}. In 6.5 and later, you 
+can also use {metricbeat} to collect and ship data about {es}. 
```

Comment:
For the metricbeat alternative, I think it would be useful to convey that metricbeat ships directly to the monitoring cluster. So what about changing the first sentence to something like this?

> Alternatively, in 6.4 and later you can use {metricbeat} to collect monitoring data about {kib} and ship it directly to the monitoring {es} cluster rather than routing it through the production {es} cluster.

Obviously, feel free to edit as you see fit!